Source/WebCore/page/scrolling/ScrollingCoordinatorTypes.h

```diff
@@ -116,18 +116,21 @@ enum class ViewportRectStability {
 
 enum class ScrollRequestType : uint8_t {
     PositionUpdate,
+    DeltaUpdate,
     CancelAnimatedScroll
 };
 
 struct RequestedScrollData {
     ScrollRequestType requestType { ScrollRequestType::PositionUpdate };
     FloatPoint scrollPosition;
+    FloatSize scrollDelta { };
```

Comment:
No need for {}. Since `scrollPosition` and `scrollDelta` should never both be set, perhaps this should use a `std::variant<>`.

Reply:
Fixed to just be scrollPosition.